pkg/controllers/machine/lifecycle/registration.go

```diff
@@ -69,17 +69,27 @@ func (r *Registration) Reconcile(ctx context.Context, machine *v1alpha5.Machine)
 	metrics.MachinesRegisteredCounter.With(prometheus.Labels{
 		metrics.ProvisionerLabel: machine.Labels[v1alpha5.ProvisionerNameLabelKey],
 	}).Inc()
+	metrics.NodesCreatedCounter.With(prometheus.Labels{
```

Comment:
Is it intended to be NodesCreatedCounter here? AFAIK, it should be MachineCreatedCounter, but this goes in line with my comment that we should have metrics for both.

Reply:
We do have both. We only know the Node is created when it registers with the cluster